test/tests/performance/pbs_preemptperformance.py

```diff
@@ -189,22 +189,21 @@ def test_insufficient_resc_non_cons(self):
 
         self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
 
-        a = {ATTR_l + '.select': '1:ncpus=1:qlist=list1'}
-        for _ in range(3200):
-            j = Job(TEST_USER, attrs=a)
-            j.set_sleep_time(3000)
-            self.server.submit(j)
-
         a = {ATTR_l + '.select': '1:ncpus=1:qlist=list2'}
         j = Job(TEST_USER, attrs=a)
         j.set_sleep_time(3000)
 
         # Add qlist to the resources scheduler checks for
         self.scheduler.add_resource('qlist')
-        self.server.manager(MGR_CMD_UNSET, SCHED, 'preempt_sort',
-                            runas=ROOT_USER)
 
         jid = self.server.submit(j)
+
```

Comment:
Thanks for fixing these tests. Do you know what numbers do these tests report before and after the change?

Reply:
I'd put a sleep 1 after this submit and the for loop below.  This forces the list2 job to be the one running the longest.